spread.yaml

```diff
@@ -284,6 +284,7 @@ restore: |
     fi
 
     rm -f $SPREAD_PATH/snapd-state.tar.gz
+    rm -rf $GOPATH/*
```

Comment:
Ok, but in that case, should I fix the other places where GOPATH is used as a path? These are the references to that:
- tests/lib/apt.sh
- tests/lib/prepare-project.sh
- tests/lib/prepare.sh
- tests/main/debs-have-built-using/task.yaml
- tests/unit/gccgo/task.yaml
- tests/upgrade/basic/task.yaml

Perhaps I could create a different branch for this